homeassistant/components/zwave_js/api.py

```diff
@@ -83,6 +83,13 @@ async def async_get_entry_func(
                 msg[ID], ERR_NOT_FOUND, f"Config entry {entry_id} not found"
             )
             return
+
+        if entry.state != ENTRY_STATE_LOADED:
+            connection.send_error(
+                msg[ID], ERR_NOT_FOUND, f"Config entry {entry_id} not loaded"
```

Comment:
Is it ok to reuse `ERR_NOT_FOUND` for this case?

Reply:
This is the message in the browser console:
```
Uncaught (in promise)
Object
code: "not_found"
message: "Config entry 62e98d4a997891f46a10d1837e633cda not loaded"
```

Reply:
I would make a new one that says not_loaded